setup.cfg

```diff
@@ -1,38 +0,0 @@
-# pytest pycodestyle configuration
-[tool:pytest]
-# Function starting with the following pattern are considered for test cases.
-python_functions=test_
-
-# pytest pycodestyle configuration
-codestyle_max_line_length = 88
-codestyle_ignore = E302 W503 E203 E501 E265 E402 E251 E211
```

Comment:
I think these configurations for `pytest` are missing from `pyproject`

Reply:
but the thing is that these were invalid. They are pycodestyle configuration that should be for flake8, not pytest 🤔 I think these were kind of legacy config that we kept along and never updated.

Reply:
Gotcha 👍🏻